client/src/routes/automation/BlocklyComponent.svelte

```diff
@@ -50,26 +50,30 @@
         );
 
         const rootBlock: string =
-            '<xml><block type="automation" deletable="false" movable="false"></block></xml>';
+            '<xml><block type="esidom_automation" deletable="false" movable="false"></block></xml>';
         Blockly.Xml.domToWorkspace(Blockly.Xml.textToDom(rootBlock), workspace);
 
         blocklyService = new BlocklyService(toolbox, workspace);
 
-        entityPromise = BlocklyService.createEntities();
+        entityPromise = BlocklyService.initBlockly();
     });
 </script>
 
 <div>
     {#await entityPromise}
-        <p>Chargement de blockly ...</p>
+        <p>Chargement de Blockly ...</p>
         <div id="blocklyDivHide" />
     {:then}
         <p>
             <button on:click={blocklyService.convertToBlock()}>Convert the
```

Comment:
I believe you shouldn't call the function convertToBlock() here, the onclick attribute only needs a reference to the function.

Reply:
Please see this PR: https://github.com/SKAVALT-io/esidom/pull/25#discussion_r575419068